src/desktop/analytics/main_layout.js

```diff
@@ -23,6 +23,7 @@ const excludedRoutes = [
   "/collect(.*)",
   "/collection(.*)",
   "/collections(.*)",
+  "/feature(.*)",
```

Comment:
I forget was there another side to this coin?

Reply:
Here's the other side: 

https://github.com/artsy/force/blob/master/src/v2/Artsy/Router/buildClientApp.tsx#L73-L74